lib/Runtime/PlatformAgnostic/Platform/Common/Intl.cpp

```diff
@@ -46,8 +46,6 @@ typedef uint64_t uint64;
 
 #define U_STATIC_IMPLEMENTATION
 #define U_SHOW_CPLUSPLUS_API 1
-#pragma warning(push)
-#pragma warning(disable:4995) // deprecation warning
```

Comment:
how did this get resolved?

Reply:
I have no idea -- @MSLaguana asked what it was for in an earlier pull and i had no idea so i removed it to see and didnt find anything.